src/presentation/http/router/noteSettings.test.ts

```diff
@@ -281,4 +281,73 @@ describe('NoteSettings API', () => {
 
     test.todo('Return 403 when user authorized, but not member of the team');
   });
+
+  describe('PATCH /note-teams/new-role/:newRole', () => {
+    test('If we want to change role to "write", in database it will be stored as number : 1', async () => {
+      await global.api?.fakeRequest({
+        method: 'PATCH',
+        headers: {
+          authorization: `Bearer ${global.auth(1)}`,
+        },
+        url: '/note-settings/new-role/write',
+        body: {
+          'userId': 1,
+          'noteId': 2,
+        },
+      });
+
+      const team = await global.api?.fakeRequest({
+        method: 'GET',
+        headers: {
+          authorization: `Bearer ${global.auth(1)}`,
+        },
+        url: '/note-settings/Pq1T9vc23Q/team',
+      });
+
+      if (team?.json() !== undefined) {
+        expect(team?.json()).toStrictEqual([
+          {
+            'id': 2,
+            'noteId': 2,
```

Comment:
we should not expose internal ids. I'm not sure that we need to to return noteId and userId at all, since they are used as request params, so client already knows them

Reply:
i guess, that it should be fixed in different PR, because it is different route